api.bs

```diff
@@ -921,6 +932,120 @@ To <dfn>deduct privacy budget</dfn> given a [=privacy budget key=] |key|,
 1.  Return whether |newValue| is greater than or equal to 0.
 
 
+### Epoch Start Store ### {#s-epoch-start}
+
+An [=epoch=] starts at a randomly-selected time
+for each [=site=].
+This ensures that any bias in aggregates
+that arises from a [=privacy budget=] reaching zero
+is averaged across all the contributions from all [=users=].
+
+The <dfn>epoch start store</dfn> is a [=map=] keyed by [=site=]
+with values that are [=moments=].
+
+To <dfn>get the current epoch</dfn>
+given an [=environment settings object=] |settings|,
+a [=site=] |site|,
+and [=moment=] |now|:
+
+1.  Let |period| be the [=duration=] of one [=epoch=] (i.e., 86400 seconds).
```

Comment:
```suggestion
1.  Let |period| be the [=duration=] of one [=epoch=].
```
It might be better to avoid duplicating the actual `86400 second` value here, in case it's changed at some point.